tools/snippet-testing/language_handler/java.rb

```diff
@@ -20,7 +20,7 @@ def text_with_specific_replacements(file_content)
     end
 
     def text_with_example_class_name(file_content)
-      file_content.gsub(/^public class\s.+\s*{\s*$/, "public class #{TEST_CLASS_NAME} {")
+      file_content.gsub(/^public class\s[^\s{]+/, "public class #{TEST_CLASS_NAME}")
```

Comment:
It seems to work and it's more concise, I'll add it. Good catch Wellington
And in both cases the spaces after the curly brace are irrelevant to the replace, so it should work anyway.